.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "Carthage/Checkouts/CryptoSwift"]
 	path = Carthage/Checkouts/CryptoSwift
-	url = https://github.com/krzyzanowskim/CryptoSwift.git
+	url = https://github.com/xmkevinchen/CryptoSwift.git
```

Comment:
I don't want to move to a third party fork, but rather stick to the original CryptoSwift.